docs/index.md

```diff
@@ -130,9 +130,19 @@ Usage: (Options preceded by an asterisk are required) [options]
        The amount of Disk resource to allocate to the elasticsearch instance
        (MB).
        Default: 1024.0
+    --elasticsearchExecutorCpu
+       The amount of CPU resource to allocate to the elasticsearch executor.
+       Default: 0.1
+    --elasticsearchExecutorRam
+       The amount of ram resource to allocate to the elasticsearch executor
+       (MB).
+       Default: 32.0
```

Comment:
When regenerating the CLI docs, there were some that were not included from previous commits.